PARAMS:
  img_win_id: 111
  rec_win_id: 112
  plt_win_id: 222


TRAIN:
  learning_rate: 0.001
  num_epochs: 50
  batch_size: 512

DIR:
  base_dir: "models"
  reconsLoss: "reconsLoss.csv"
  saved_models: "saved_models"

TEST:
  gpu: False
  enc_dec_model: "epoch_30.pt"



